bedrock/mozorg/templates/mozorg/internet-health/index.html

```diff
@@ -4,7 +4,7 @@
 
 {% extends "base-pebbles.html" %}
 
-{% add_lang_files "mozorg/internet-health/index" %}
+{% add_lang_files "mozorg/internet-health/shared" %}
```

Comment:
You can use this branch to avoid creating the files by hand 
https://github.com/mozilla-l10n/www.mozilla.org/tree/Jun8